src/ui/molecules/bridge.transfer/bridge.transfer.fee.js

```diff
@@ -23,19 +23,34 @@ export const BridgeTransferFee = ({
   }, [amount, token]);
   // Getting Fees - END
 
+  useEffect(() => {
+    if (fee && amount) {
+      setFee(null);
+    }
+  }, [token]);
+
   var formatter = new Intl.NumberFormat("en-US", {
     style: "currency",
     currency: "USD",
   });
 
   function formatConversionOutput() {
+    const formattedFee = fee && fee != "" ? fee : "0";
+    console.log("Formatted FEE: " + formattedFee);
+
     switch (token) {
       case "USDC":
-        return fee;
+        return formattedFee;
       case "ETH":
-        return fee * ethers.utils.formatUnits(eth_usd, 6);
+        return ethers.utils.formatUnits(
+          ethers.utils.parseEther(formattedFee).mul(eth_usd),
+          24
```

Comment:
In general I don't like having magic numbers, especially without context.  What is 24 referencing here?